content/kubermatic/main/architecture/requirements/cluster-requirements/_index.en.md

```diff
@@ -50,27 +50,29 @@ If you have more than one network adapter, and your Kubernetes components are no
 
 ## Check Required Ports
 
-The tables below list the ports that need to be open for communication, e.g. by allowing them via a firewall.
-
-### Master Cluster Master Node(s)
-
-| Protocol | Direction | Port Range | Purpose                 |
-|----------|-----------|------------|-------------------------|
-| TCP      | Inbound   | 6443*      | Kubernetes API server   |
-| TCP      | Inbound   | 2379-2380  | etcd server client API  |
-| TCP      | Inbound   | 10250      | kubelet API             |
-| TCP      | Inbound   | 10251      | kube-scheduler          |
-| TCP      | Inbound   | 10252      | kube-controller-manager |
-| TCP      | Inbound   | 10255      | Read-only kubelet API   |
-
-### Worker Node(s) & User Cluster Worker Nodes
-
-| Protocol | Direction | Port Range  | Purpose               |
-|----------|-----------|-------------|-----------------------|
-| TCP      | Inbound   | 10250       | kubelet API           |
-| TCP      | Inbound   | 10255       | Read-only kubelet API |
-| TCP      | Inbound   | 30000-32767 | NodePort Services**   |
-
-** Default port range for [NodePort Services](https://kubernetes.io/docs/concepts/services-networking/service/).
+Please ensure that all nodes in a user cluster can communicate without restriction to ensure functionality of CNI/CSI and Kubernetes itself.
+In addition, user cluster nodes must be able to connect to the Seed cluster's nodeport-proxy. This depends on the [expose strategy]({{< ref "../../../tutorials-howtos/networking/expose-strategies" >}}.
+
+It is recommended to make yourself familiar with the [concept of networking]({{< ref "../../../architecture/concept/kkp-concepts/networking" >}}) in KKP.
+
+| Source                  | Destination                   | Expose Strategy | Ports             | Purpose                                              |
+|-------------------------|-------------------------------|-----------------|-------------------|------------------------------------------------------|
+| KKP Users               | Master Ingress Controller     | Any             | 443*              | Access to KKP Dashboard                              |
+| KKP Operator            | Master cluster Kubernetes API | Any             | 6443*             | Operator access                                      |
+| KKP Operator            | Seed cluster Kubernetes API   | Any             | 6443*             | Operator access                                      |
```

Comment:
For this - and below - I think we should specify the source as the "physical" instance that makes the network connections; and in this case, it would be the "Master cluster nodes". Understanding where the KKP Operator or Kubermatic API run requires architectural knowledge that the target audience of this table (network engineers) might not have. WDYT?

Reply:
We can add a column indicating in what cluster it runs

Reply:
I would like to keep the table as simple as possible, so maybe the "Source" column mentions the "node / hardware" source and the comment/description includes component information like this?

Reply:
Honestly I would leave it as it is. This information is detailed in the networking concept page that is linked above.

Reply:
I don't agree with that - imagine you're a network engineer at a user/customer. You get a link to this page to set up firewall rules accordingly. You probably only have a superficial understanding of the KKP architecture and you don't really care either.

We're mixing components and "real" sources here because later on, we mention "user cluster nodes" - but if we followed the table's pattern, it would need to be "kubelet, konnectivity, CSI drivers" or something like that.

This table makes sense to us because we know where components will run, but the networking concept page does not explain where the operator runs and I don't think we should make it necessary to get knowledge from other pages in the docs when we can make this piece of information self-contained.

Reply:
Okay I have no idea how to build it into the table though :D Do you have an idea?